server/game/basecard.js

```diff
@@ -52,6 +52,9 @@ class BaseCard {
         this.parseKeywords(cardData.text || '');
         this.parseTraits(cardData.traits || '');
         this.setupCardAbilities(AbilityDsl);
+
+        this.factions = {};
+        this.factions[cardData.faction_code] = 1;
```

Comment:
Should toLowerCase the faction here for consistency. Or better yet, use the addFaction method.